compute_tools/src/config.rs

```diff
@@ -74,10 +74,17 @@ pub fn write_postgres_conf(
     }
 
     // Locales
-    writeln!(file, "lc_messages='C.UTF-8'")?;
-    writeln!(file, "lc_monetary='C.UTF-8'")?;
-    writeln!(file, "lc_time='C.UTF-8'")?;
-    writeln!(file, "lc_numeric='C.UTF-8'")?;
+    if cfg!(target_os = "macos") {
```

Comment:
Nit, should “macos” be a string? Is there a const or enum that would be a little safer?

Reply:
yes, unfortunately, that's how Rust works now

Reply:
I'm not aware of any Rust enum for cfg checks. I'll look around.

Reply:
Apparently this exists in a crate, but it isn't worth adding.